chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -301,12 +301,12 @@ impl PeerManagerActor {
                 continue;
             }
             let key = edge.get_pair();
-            if !self.routing_table_view.is_local_edge_newer(&key, edge.nonce) {
+            if !self.routing_table_view.is_local_edge_newer(&key, edge.nonce()) {
                 continue;
             }
             self.routing_table_view
                 .local_edges_info
-                .insert((edge.key.0.clone(), edge.key.1.clone()), edge.clone());
+                .insert((edge.key().0.clone(), edge.key().1.clone()), edge.clone());
```

Comment:
Isn't this just 

```suggestion
                .insert(edge.key().clone(), edge.clone());
```

? If this is corret, there are a couple more places where we want this. 

Also, looking at this, maybe it makes sense to define (in a separte PR, if at all)

```rust
struct EdgeMap {
  raw: HashMap<(PeerId, PeerId), Edge>
}
```

? This'll make it easer to get rid of key duplication.